scripts/ethereum/deployment/src/deploy_contracts.ts

```diff
@@ -40,6 +41,7 @@ export async function main(args: ReadonlyArray<string>): Promise<void> {
   // Order matters to get reproducible contract addresses
   const deploymentJobs: ReadonlyArray<DeploymentJob> = [
     { name: "AshToken", contract: new AshToken(eth) },
+    { name: "AtomicSwapEther", contract: new AtomicSwapEther(eth) },
```

Comment:
This must come after TrashToken, otherwise the TrashToken is deployed using a different nonce, which changes the contract address